CRM/Core/PrevNextCache/Sql.php

```diff
@@ -74,4 +74,105 @@ public function fillWithArray($cacheKey, $rows) {
     return TRUE;
   }
 
+  /**
+   * Save checkbox selections.
+   *
+   * @param string $cacheKey
+   * @param string $action
+   *   Ex: 'select', 'unselect'.
+   * @param array|int|NULL $cIds
+   *   A list of contact IDs to (un)select.
+   *   To unselect all contact IDs, use NULL.
+   */
+  public function markSelection($cacheKey, $action, $cIds = NULL) {
```

Comment:
$cIds is updated to the (better) $ids in the Interface & is inconsistent

Reply:
Agree `$ids` is nicer. I've added a revision to my big branch to use `$ids`. There are some other cleanups which touch that file, so it'll show up towards the end of the queue.